netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -49,6 +53,11 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 2.14
+ * @todo #750:30min Solve the puzzle for build mysteriously failing for
+ *  this task citing a NullPointerException (See qulice issue #608 raised for
+ *  the same) and then remove TkStart.java from qulice exceptions.
+ * @todo #750:30min Solve the puzzle for needing an Object array in the
```

Comment:
@prahladyeri I just had an idea - since we are importing `Commons-Lang3` we can use [Pair<Date, Inbox>](https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/tuple/Pair.html). Please update the code accordingly, I don't think we will need a puzzle for this.